test/unit/dialects/postgres/query-generator.test.js

```diff
@@ -1331,5 +1331,53 @@ if (dialect.startsWith('postgres')) {
         });
       });
     });
+
+    describe('Custom Schema Functions', () => {
+      beforeEach(function () {
+        this.queryGenerator = new QueryGenerator({
```

Comment:
create a new sequelize instance instead, with the option set (see other comments on how to create a new test instance)

Reply:
fixed